Pilot Data Exploration.R

```diff
@@ -24,46 +24,213 @@ if(pull_fresh_dataset) {
+                            pfs_completers$pi_pfs_7)
+mean(as.matrix(pi_pfs_all_items))
+sd(as.matrix(pi_pfs_all_items))
+
+#Individual item scores
+mean(pfs_completers$pi_pfs_1)
+sd(pfs_completers$pi_pfs_1)
+mean(pfs_completers$pi_pfs_2)
+sd(pfs_completers$pi_pfs_2)
+mean(pfs_completers$pi_pfs_3)
+sd(pfs_completers$pi_pfs_3)
+mean(pfs_completers$pi_pfs_4)
+sd(pfs_completers$pi_pfs_4)
+mean(pfs_completers$pi_pfs_5)
+sd(pfs_completers$pi_pfs_5)
+mean(pfs_completers$pi_pfs_6)
+sd(pfs_completers$pi_pfs_6)
+mean(pfs_completers$pi_pfs_7)
+sd(pfs_completers$pi_pfs_7)
+
+## Hopelessness
+plot(b_pi_completers$b_bhs_sum, b_pi_completers$pi_bhs_sum)
+t.test(b_pi_completers$b_bhs_sum, b_pi_completers$pi_bhs_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_bhs_sum, pi_bhs_sum) %>%
+  mutate(bhs_sum_change = pi_bhs_sum - b_bhs_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = bhs_sum_change_mean / bhs_sum_change_sd,
+         d_av = bhs_sum_change_mean / ((b_bhs_sum_sd + pi_bhs_sum_sd) / 2))
+
+## Agency - this is the subscale we messed up!
+plot(b_pi_completers$b_shs_agency_sum, b_pi_completers$pi_shs_agency_sum)
+t.test(b_pi_completers$b_shs_agency_sum, b_pi_completers$pi_shs_agency_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_shs_agency_sum, pi_shs_agency_sum) %>%
+  mutate(shs_agency_sum_change = pi_shs_agency_sum - b_shs_agency_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = shs_agency_sum_change_mean / shs_agency_sum_change_sd,
+         d_av = shs_agency_sum_change_mean / ((b_shs_agency_sum_sd + pi_shs_agency_sum_sd) / 2))
+
+## Functionality Appreciation Scale
+plot(b_pi_completers$b_fas_sum, b_pi_completers$pi_fas_sum)
+t.test(b_pi_completers$b_fas_sum, b_pi_completers$pi_fas_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_fas_sum, pi_fas_sum) %>%
+  mutate(fas_sum_change = pi_fas_sum - b_fas_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = fas_sum_change_mean / fas_sum_change_sd,
+         d_av = fas_sum_change_mean / ((b_fas_sum_sd + pi_fas_sum_sd) / 2))
+
+## Body Dissatisfaction 
+plot(b_pi_completers$b_bsss_sum, b_pi_completers$pi_bsss_sum)
+t.test(b_pi_completers$b_bsss_sum, b_pi_completers$pi_bsss_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_bsss_sum, pi_bsss_sum) %>%
+  mutate(bsss_sum_change = pi_bsss_sum - b_bsss_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = bsss_sum_change_mean / bsss_sum_change_sd,
+         d_av = bsss_sum_change_mean / ((b_bsss_sum_sd + pi_bsss_sum_sd) / 2))
 
 
+## Demographics
 
-####  Analysis  ####
-## Functional Appreciation Scale
-plot(clean_data$b_fas_sum, clean_data$pi_fas_sum)
-t.test(clean_data$b_fas_sum, clean_data$pi_fas_sum)
+# Sex assigned at birth
+b_pi_completers %>%
+  count(b_dem_sex) %>%
+  mutate(percent = (n / sum(n))*100)
```

Comment:
If you want an easy way to print percents, you can do this

```
library(scales)
percent(.05)
```

but note that it outputs a character vector (`"5%"`) not a numeric one (`.05`)